20-minimal/README.md

```diff
@@ -321,5 +321,5 @@ See also
 --------
 Dockerfile and other sources are available on https://github.com/sclorg/s2i-nodejs-container.
 In that repository you also can find another versions of Node.js environment Dockerfiles.
-Dockerfile for CentOS is called `Dockerfile`, Dockerfile for RHEL7 is called `Dockerfile.rhel7`,
+Dockerfile for RHEL7 is called `Dockerfile.rhel7`,
```

Comment:
CentOS Stream images are missing

Reply:
Yes, I did not include them for purpose. I think people will find them even without mentioning this information explicitly.

Reply:
When we delete the CentOS7 Dockerfiles, that are missing the suffix with distribution the naming of the Dockefiles becomes more straightforward.

Reply:
Also when I was going through the repositories when deprecating c8s, I noticed that many repositories contain only minimal information about CentOS Stream images, so this would not be the only container image repository in this state.

Reply:
It does not make sense to have Dockerfile for rhel7, Dockerfile for rhel8, and not for Centos Stream images.

It can lead to misunderstandings, why they are not present.

Reply:
fixed.